repository/repository-web/src/main/resources/application-eclipse.properties

```diff
@@ -0,0 +1,50 @@
+server.port=8080
+server.host=vorto-repo
+server.contextPath=/
+server.admin=${list of userids with ; as delimiter}
+server.upgrade.gdpr=false
+server.config.authenticatedSearchMode=false
+
+spring.profiles.active=eclipse
+spring.config.location=config
+
+spring.datasource.url=jdbc:mysql://${database:port}/vorto
+
+spring.datasource.username=${datasource_user_name}
+spring.datasource.password=${datasource_user_password}
+
+multipart.maxFileSize=128MB
+multipart.maxRequestSize=128MB
+
+github.oauth2.enabled=true
+github.oauth2.client.clientId=${github_clientId}
+github.oauth2.client.clientSecret=${github_clientSecret}
+github.oauth2.client.accessTokenUri=https://github.com/login/oauth/access_token
+github.oauth2.client.userAuthorizationUri=https://github.com/login/oauth/authorize
+github.oauth2.client.clientAuthenticationScheme=form
+github.oauth2.client.scope=\"user:email\"
+github.oauth2.resource.userInfoUri=https://api.github.com/user
+
+eidp.oauth2.enabled=false
+eidp.oauth2.client.clientId=${eidp_clientId}
+eidp.oauth2.client.clientSecret=${eidp_clientSecret}
+eidp.oauth2.client.accessTokenUri=https://accounts.bosch.com/adfs/oauth2/token
+eidp.oauth2.client.userAuthorizationUri=https://accounts.bosch.com/adfs/oauth2/authorize
+eidp.oauth2.client.clientAuthenticationScheme=form
+eidp.oauth2.client.resource=https://vorto.eclipse.org
+eidp.oauth2.client.scope=\"user:email\"
+eidp.oauth2.resource.userInfoUri=https://accounts.bosch.com/adfs/userinfo
+eidp.oauth2.resource.preferTokenInfo=true
+eidp.oauth2.resource.logoutRedirectUrl=${eidp_logout_redirecturl}
+
+webEditor.enabled=true
```

Comment:
all webEditor.* should not be needed anymore. Please remove. Can you crosscheck ?

Reply:
okay i will check and update accordingly.